sections/semantics-document-metadata.include

```diff
@@ -963,6 +974,75 @@
           inappropriate keywords.
         </p>
 
+  : <dfn><code>referrer</code></dfn>
+
+  :: The value must be a <a>referrer policy</a>, which defines the default <a>referrer
```

Comment:
Wrapping a link is bad. Could you fix this one link so we don't line break in the middle of an link element.